### ISSUE TITLE
IOS Build Failing

### ISSUE THREAD

**Issue by SergeWilfried** [cmaycumber/config-plugin-react-native-intercom#5]
IOS Build is failing with the following error : 

```
TypeError: null is not an object (evaluating 'IntercomEventEmitter.UNREAD_COUNT_CHANGE_NOTIFICATION')
```
**Installed** 

- Expo-cli@4.4.4
- Expo SDK 43
- Device IPhone 13

**Comment by manoelcfeliciano** [comment]
Hi @SergeWilfried, please refer to the following issue:

https://github.com/cmaycumber/config-plugin-react-native-intercom/issues/1

I was having the same problem, and fixed by following the suggestion made there.

**Comment by Chris-Beckett** [comment]
> Hi @SergeWilfried, please refer to the following issue:
> 
> #1
> 
> I was having the same problem, and fixed by following the suggestion made there.

Were you have the issue when trying to run through the `npm run android` via `Metro` and the solution is to using `expo run:android` instead?

**Comment by cmaycumber** [comment]
Going to close this because it seems solved in #1.
